vtty.c

```diff
@@ -679,27 +791,27 @@ static struct miscdevice* vtmx_miscdev = NULL;
 
 static int __init vtty_init(void)
 {
-  static struct miscdevice vtmx_miscdev_data;
-  int ret = 0;
-  unsigned long driver_flags = 0;
+	static struct miscdevice vtmx_miscdev_data;
```

Comment:
I try really hard every time. But what is the lesser evil: `push --force` after `commit --amend` oder an ugly history with lots of merge commits after `commit --amend`?
I promise, I try harder...

Reply:
Personally I don't see any evil in doing push --force, amend, rebase and so on - as long as it happens on a personal development branch. When code gets pushed to a shared repository, then it's a different matter.

If I understand correctly, the typical pattern for having a Linux patch series accepted is to submit something equivalent to a chain of commits to a mailing list. Then the chain gets reviewed over and over, and the patches (commits) are re-submitted after applying changes. This goes back-and-forth until the patch series is accepted. So it's normal that the submitter would make heavy use of these "evil" tools, instead of stacking commits on top of commits, making the patch series more and more difficult to review.

Reply:
I already accepted internally to craft a new patch series for my pull request(s), so then those amended commits will disappear naturely. I'll work on those during the next days, so, please, standby for amended pull requests.